docs/components/scenarios.md

```diff
@@ -1,3 +1,11 @@
+---
+layout: default
+title: Projects.json
+nav_order: 1
+parent: Configure GrimoireLab
+grand_parent: Components
+---
+
```

Comment:
Please let me know if I need to update the front matter to create the dropdown as mentioned in the PR description.

Reply:
In order to have the dropdown as per the PR description, we'll need one `index.md` file titled `Components` to act as the parent section. It needs to have the following config.
```
layout: default
title: Components
nav_order: 2
permalink: /components
has_children: true
has_toc: true
```

Then, the three dropdown element (`component.md`, `scenario.md` and `workflow.md`) should all the have following in the beginning of their respective md file,

```
---
layout: default
title: <replace with proper title, e.g Workflow>
nav_order: <replace with order number of this file in dropdown>
parent: Components
---
```

Reply:
Done!